newjson.py

```diff
@@ -0,0 +1,89 @@
+import json
+import time
+
+phonebook = {}
+
+with open("json1.json") as rfile:
+    phonebook = json.load(rfile)
+    print(phonebook)
+while True:
+    a = input("Write a command:")
+
+    if a == "Add":
+        b = input("Write name:")
+        c = input("Write number:")
+
+    with open("json1.json", "w") as wfile:
+        phonebook[f'{b}'] = f'{c}'
+        json.dump(phonebook, wfile)
+        print(phonebook)
+
+    if a == "Delete":
+        name = input("Write name:")
+        with open("json1.json") as dfile:
+            phonebook = json.load(dfile)
+            if name in phonebook:
+                del phonebook[name]
+            with open("json1.json", "w") as wwfile:
+                json.dump(phonebook, wwfile)
+
+    if a == "List":
+        with open("json1.json") as refile:
+           abc = refile.read()
+           abcd = json.loads(abc)
+           print(abcd)
+
+
+
+def my_decorator(func):
+    def wrap(*args, **kwargs):
+        return func(*args, **kwargs)
+    return wrap
+
+@my_decorator
+def my_func():
+    file = open("time.txt", "r+")
+    file_content = file.read()
+    file_content = "my_func" + time.strftime(" %H" ":" "%M") + file_content
+    file.write(file_content)
+my_func()
+
+
+class MyCustomException(Exception):
+    with open('error.txt', 'r+') as file:
+        text = file.read()
+        text = text + f'{Exception}'+ time.strftime(" %H"":""%M")
+        file.write(text)
```

Comment:
I suggest you wanted to do something like that

```suggestion
    def __init__(self, message="Custom exception is occurred"):
        self.message = message
        self.logging()
        super().__init__(self.message)

    def logging(self):
        with open("error.txt", "a+") as file:
            text = f"Raised MyCustomException with massage {self.message}. Time: {datetime.now()}"
            file.write(text)
```

I didn't test this code, so you are welcome to play with it